src/components/AudioPlayer/AudioProgress/AudioProgress.container.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from 'react';
+import propTypes from 'prop-types';
+
+import {
+  convertDurationToProgress,
+  convertInputToProgress,
+  convertProgressToDuration,
+} from './songProgress';
+
+import AudioProgressComponent from './AudioProgress.component';
+
+const AudioProgressContainer = ({ audioRef }) => {
+  const [progress, setProgress] = useState(0);
+  const [isBeingDragged, setIsBeingDragged] = useState(false);
+
+  const handleDragging = () => setIsBeingDragged(true);
+
+  const handleStopDragging = () => {
+    const { duration } = audioRef.current;
+    const nextProgress = convertProgressToDuration(progress, duration);
+    // eslint-disable-next-line no-param-reassign
```

Comment:
Remember to fix your code instead of adding these overriding comments.

Reply:
I kept it because i didn't find another way to change the current time of the song, since its an element of the DOM.